addons/main/script_version.hpp

```diff
@@ -1,4 +1,4 @@
 #define MAJOR 1
 #define MINOR 1
 #define PATCHLVL 0
-#define BUILD 4
+#define BUILD 4
```

Comment:
Newline?

Reply:
The new line is stripped by the makefile, I’ll see if I can append `\n`.

Reply:
You are appending a newline at the others, why not the last one.